meta/parse.pl

```diff
@@ -120,6 +120,10 @@ sub ProcessTagType
 
     return $val if $val =~ /^sai_pointer_t sai_\w+_notification_fn$/;
 
+    return $val if $val =~ /^sai_pointer_t sai_\w+_register_read_fn$/;
```

Comment:
not sure if this make sense here to add every function, as mentioned before all pointers in switch are a notification type and follow convention of notifications. I would suggest to change names to be named as notifications scheme. There is logic later on which generates automatic structures from that and enum values.

As asked before, why we have those exact functions here if we have same functions in API structure ?
Can you describe purpose of those functions on API structure and on attributes ?

Reply:
i reformatted this in #1008